CHANGELOG.md

```diff
@@ -12,6 +12,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - added new method `Grid::delete_orders` and the corresponding switch
   `--delete-orders` in the subcommand `write` of the CLI
 
+### Changed
+
+- renamed `pineappl.lumi.rs` to `pineappl.channel.rs`
```

Comment:
prefix all of your items with `Python API:` to make clear that this concerns it and not the Rust API.

Reply:
@cschwan correct me if I'm wrong, but I believe what he meant is
```
- PythonAPI: dropped top-level Python interface layer
- PythonAPI: blub
```

Reply:
... with a space between 'Python' and 'API' though.